<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QWidget" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Whisper - Registration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/Icons/Resource/Icon/ic_LOGO_64.png</normaloff>:/Icons/Resource/Icon/ic_LOGO_64.png</iconset>
  </property>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>82</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(32, 101, 144);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>474</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 68px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Other/Resource/botton.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>142</y>
     <width>431</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,140);
border-radius: 8px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="register_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>92</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Grande</family>
     <pointsize>28</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
color: rgb(151, 184, 201);
background-color: rgb(255, 255, 255,0);
</string>
   </property>
   <property name="text">
    <string>REGISTRATION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>22</y>
     <width>651</width>
     <height>601</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Other/Resource/background.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="accept_button">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(255,255, 200,80);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="text">
    <string>ACCEPT</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sex">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>238</y>
     <width>32</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sex:</string>
   </property>
  </widget>
  <widget class="QLabel" name="age_2">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>284</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="city">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>330</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>City:</string>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>379</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="username">
   <property name="geometry">
    <rect>
     <x>152</x>
     <y>185</y>
     <width>88</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_user_name">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>185</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(22, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="maxLength">
    <number>14</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="sex_person">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>238</y>
     <width>100</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 188, 199);</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Unknown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Man</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Woman</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="age">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>284</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(161, 188, 199);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_city">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>330</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(22, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="enter_password">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>379</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(22, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="maxLength">
    <number>14</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Error_label">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>211</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 71, 0);</string>
   </property>
   <property name="text">
    <string>Error.. Name already exists.</string>
   </property>
  </widget>
  <widget class="QLabel" name="Error_label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(214, 71, 0);</string>
   </property>
   <property name="text">
    <string>Error.. Enter username and password.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="come_back">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>485</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
border-width: 1px;
border-style: solid;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Resource/Icon/ic_back.png</normaloff>:/Icons/Resource/Icon/ic_back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Other/Resource/header.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="turn_window">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Resource/Icon/ic_turn_brown.png</normaloff>:/Icons/Resource/Icon/ic_turn_brown.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="close_window">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>3</y>
     <width>21</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Resource/Icon/ic_close_brown.png</normaloff>:/Icons/Resource/Icon/ic_close_brown.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="confirm">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm pas:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_confirm">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>420</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 5px;
background-color: rgb(22, 0, 200,10);
border-width: 1px;
     border-style: solid;
     border-color:  rgb(255, 255, 255, 200);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>label_6</zorder>
  <zorder>register_2</zorder>
  <zorder>label_10</zorder>
  <zorder>accept_button</zorder>
  <zorder>sex</zorder>
  <zorder>age_2</zorder>
  <zorder>city</zorder>
  <zorder>password</zorder>
  <zorder>username</zorder>
  <zorder>sex_person</zorder>
  <zorder>age</zorder>
  <zorder>enter_user_name</zorder>
  <zorder>enter_city</zorder>
  <zorder>enter_password</zorder>
  <zorder>Error_label</zorder>
  <zorder>Error_label_2</zorder>
  <zorder>come_back</zorder>
  <zorder>label_9</zorder>
  <zorder>turn_window</zorder>
  <zorder>close_window</zorder>
  <zorder>confirm</zorder>
  <zorder>password_confirm</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
